lib/src/credentials/linked_data/ld_base_suite.dart

```diff
@@ -144,14 +159,23 @@ abstract class LdBaseSuite<VC extends DocWithEmbeddedProof, Model extends VC>
         final cryptosuite = proof['cryptosuite'] as String?;
         switch (cryptosuite) {
           case 'ecdsa-rdfc-2019':
-            return DataIntegrityEcdsaVerifier(issuerDid: issuerDid);
+            return DataIntegrityEcdsaVerifier(
+              issuerDid: issuerDid,
+              customDocumentLoader: customDocumentLoader ?? _noOpLoader,
```

Comment:
```suggestion
              customDocumentLoader: customDocumentLoader,
```

`customDocumentLoader` is optional and it is not required to provide the fallback if it is null. The fallback is already being assumed [here](https://github.com/affinidi/affinidi-ssi-dart/blob/9a6dfe06afa79415c627ffaf84b1dba2e59ccf0f/lib/src/credentials/proof/embedded_proof_suite.dart#L57). So we don't need to define and use `_noOpLoader` here.